agent-discoveries-frontend/src/Views/ProfilePage/ProfilePage.tsx

```diff
@@ -1,7 +1,62 @@
 import React from 'react'
+import './ProfilePage.css'
+import Title from '../../Components/Title/title'
+import MessageBoard from '../../Components/MessageBoard/messageBoard'
+import AgentsList from '../../Components/AgentsList/agentsList'
+import PostMessage from '../../Components/PostMessage/postMessage'
 
-const ProfilePage = () => {
-  return <></>
+function Header() {
+  return (
+    <header className="banner">
+      <button>Edit Profile</button>
+      <img src="" alt="" />
+      <p>Location</p>
+      <p>Time</p>
+    </header>
+  )
 }
 
-export default ProfilePage
+function AccountInfo() {
+  return (
+    <section className="column column1">
```

Comment:
I'd consider a different name for "column1", it's not obvious how it's meant to differ from "column"